cachier/core.py

```diff
@@ -234,23 +192,16 @@ def cachier(
             cache_dir=cache_dir,
             separate_files=separate_files,
             wait_for_calc_timeout=wait_for_calc_timeout,
-            default_params=_default_params,
         )
     elif backend == "mongo":
-        if mongetter is None:
```

Comment:
Are you sure you can delete this check? What if somebody explicitly provides `backend="mongo"` and `mongetter=None`?

Reply:
Ok I see this was relegated to the mongo core. Nice.

Reply:
It was moved to class not deleted